tests/tracer/test_propagation.py

```diff
@@ -2918,6 +2932,60 @@ def test_span_links_set_on_root_span_not_child(fastapi_client, tracer, fastapi_t
             _HTTP_HEADER_TRACESTATE: "",
         },
     ),
+    (
+        "only_baggage",
+        [
+            _PROPAGATION_STYLE_BAGGAGE,
+        ],
+        {
+            "baggage": {"foo": "bar"},
+        },
+        {
+            "baggage": "foo=bar",
+        },
+    ),
+    (
+        "baggage_and_datadog",
+        [
+            PROPAGATION_STYLE_DATADOG,
+            _PROPAGATION_STYLE_BAGGAGE,
+        ],
+        {
+            "trace_id": VALID_DATADOG_CONTEXT["trace_id"],
+            "span_id": VALID_DATADOG_CONTEXT["span_id"],
+            "baggage": {"foo": "bar"},
+        },
+        {
+            HTTP_HEADER_TRACE_ID: "13088165645273925489",
+            HTTP_HEADER_PARENT_ID: "8185124618007618416",
+            "baggage": "foo=bar",
+        },
+    ),
+    (
+        "baggage_order",
+        [
+            _PROPAGATION_STYLE_BAGGAGE,
+            PROPAGATION_STYLE_DATADOG,
+            PROPAGATION_STYLE_B3_MULTI,
+            PROPAGATION_STYLE_B3_SINGLE,
+            _PROPAGATION_STYLE_W3C_TRACECONTEXT,
+        ],
+        {
+            "baggage": {"foo": "bar"},
+            "trace_id": VALID_DATADOG_CONTEXT["trace_id"],
+            "span_id": VALID_DATADOG_CONTEXT["span_id"],
+        },
+        {
+            "baggage": "foo=bar",
```

Comment:
In a similar way that we have  HTTP_HEADER_PARENT_ID, HTTP_HEADER_TRACE_ID, etc, could we set `_HTTP_HEADER_BAGGAGE: Literal["baggage"] = "baggage"` at the top and use that here and in the rest of the spots.